src/views/Pathways/Pathways/Edit/index.js

```diff
@@ -50,10 +51,19 @@ const PathwayEdit = ({ rightPaneLoading, actions, match, theme }) => {
     return <React.Fragment />
   }
 
+  const ViewMentorTree =() =>{
+    history.push(`/pathways/${pathwayId}/mentorTree`)
+  }
+
+
   return (
     <RightPaneWithTitle title="Edit Pathway" closeLink="/pathways">
       <FormBuilder structure={getPathwayEditFormStructure(pathway)} onSubmit={onSubmit} initialValues={pathway} submitBtnDisabled={submitBtnDisabled} />
       <Spacer height={theme.spacing(2)} />
+      <Typography variant="h5">Mentorship Tree</Typography>
+      <Spacer height={theme.spacing(1)} />
+      <Button fullWidth variant="contained" disableElevation color="primary" onClick={() => ViewMentorTree()}>View Mentor Tree</Button>
```

Comment:
`ViewMentorTree`. Two problems:

1. Only Component names should start with first letter as upper case. 
2. Secondly you should use a <Link> component instead of onClick for linking. onClick should be used when something complex needs to be done. Search for <Link> in the project and see how it can be used.

Reply:
I haven't seen it across the code, but make sure you fix this if it is done in a similar way at some other place in the code.

Reply:
changed into Component